ecr-login/cache/build.go

```diff
@@ -22,13 +22,13 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/config"
-	log "github.com/cihub/seelog"
-	homedir "github.com/mitchellh/go-homedir"
+	"github.com/mitchellh/go-homedir"
+	"github.com/sirupsen/logrus"
 )
 
 func BuildCredentialsCache(awsSession *session.Session, region string, cacheDir string) CredentialsCache {
 	if os.Getenv("AWS_ECR_DISABLE_CACHE") != "" {
-		log.Debug("Cache disabled due to AWS_ECR_DISABLE_CACHE")
+		logrus.Debug("Cache disabled due to AWS_ECR_DISABLE_CACHE")
```

Comment:
I'd argue this is a warning if this isn't intended to be used normally.

Reply:
It's an intentional option.  I don't think this should be a warning.